src/main/bash/commands/kobman-install.sh

```diff
@@ -1,102 +1,50 @@
 
 function __kob_install {
 
-#Latest version check and assignment should happen here (if latest has been released )
-
-if [ -z "${argument_[1]}" ];
-        then
-                __kobman_echo_no_colour "Invalid command : Try with --environment/-env "
-                return  
-        elif [ "${argument_[1]}" == "--environment" ] || [ "${argument_[1]}" == "-env"  ];  
-        then    
-		__kobman_validate_set_environment "${argument_[2]}"
-                if [ "$?" -eq "0" ]   
-                then
-                      	environment_value=${argument_[2]}
-			
-			case "${argument_[3]}" in
-			--version)
-                                if [[ "${argument_[5]}" == "--namespace" && $version_value != "" ]]; 
-                                then    
-                                        __kobman_setting_global_variables "${argument_[6]}" "${argument_[4]}"
-					__kobman_validate_and_set_version "${version_value}" "${namespace_value}"
-					if [ "$?" -eq "0" ];
-					then	
-						__kobman_create_environment_directory "$environment_value" "$version_value" "$namespace_value" 
-					fi	
-                                elif [[ "${argument_[5]}" == "" && $version_value != "" ]]; 
-                                then    
-                                        __kobman_setting_global_variables "${KOBMAN_NAMESPACE}" "${argument_[4]}"
-					__kobman_validate_and_set_version "${version_value}" "${namespace_value}"
-					if [ "$?" -eq "0" ];
-					then	
-						__kobman_create_environment_directory "$environment_value" "$version_value" "$namespace_value" 
-					fi	
-					
-                                else    
-                                        return  
-                                fi
-			;;
-			--namespace)
-                                __kobman_setting_global_variables "${argument_[4]}" "${KOBMAN_VERSION}" 
-				__kobman_validate_and_set_version "${version_value}" "${namespace_value}"
-				if [ "$?" -eq "0" ];
-				then	
-					__kobman_create_environment_directory "$environment_value" "$version_value" "$namespace_value" 
-				fi	
-			;;
-
-
-			"")
-                                __kobman_setting_global_variables "${KOBMAN_NAMESPACE}" "${KOBMAN_VERSION}" 
-				__kobman_validate_and_set_version "${version_value}" "${namespace_value}"
-				if [ "$?" -eq "0" ];
-				then	
-					__kobman_create_environment_directory "$environment_value" "$version_value" "$namespace_value" 
-				fi	
-			;;
-		
-			esac  
-                else
-                        __kobman_echo_red "Environemt not available ."
-                return  
-                fi
-    
-        fi
+	environment_value=$1
+ 	version_value=$2
+	namespace_value=$3 
+	__kobman_create_environment_directory "$environment_value" "$version_value" "$namespace_value" 
 }
 
 function __kobman_validate_set_environment
 {
+	echo $1 > $KOBMAN_DIR/var/current	
 	curl -sL "https://raw.githubusercontent.com/${KOBMAN_NAMESPACE}/KOBman/master/dist/list" | grep -i "$1" > /dev/null
```

Comment:
@asa1997  
Please make use common function to download script, please refer to drman usage of [__drman_secure_curl_download](https://github.com/DIDman/DRman/blob/master/src/main/bash/drman-utils.sh)